plugin/test.vim

```diff
@@ -24,6 +24,7 @@ let g:test#default_runners = {
   \ 'Perl':       ['Prove'],
   \ 'Racket':     ['RackUnit'],
   \ 'Java':       ['MavenTest'],
+  \ 'Scala':      ['Sbttest'],
```

Comment:
I think it would make more sense to have this named `SbtTest`.

Reply:
Fixed)